modules/tinymce/src/core/main/ts/api/commands/AlignCommands.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Arr, Type } from '@ephox/katamari';
+
+import { FormatVars } from '../../fmt/FormatTypes';
+import type Editor from '../Editor';
```

Comment:
```suggestion
import Editor from '../Editor';
```
We probably shouldn't be using type imports, as the types are published so this would make it incompatible with older versions of TypeScript so for now given it's not needed we're probably safer doing this (granted type only imports been around for a while now so it might be fine). If we did want this, we probably need to have a quick discussion and document what version of TS our types support.

Reply:
It was added in 3.8 two years ago but there might be legacy projects using tinymce so unsure if we can use this "new" feature. We might also consider adding a step that verifies the types with an older version of typescript so that we don't accidentally use features that might be incompatible.